src/roulette.py

```diff
@@ -126,3 +126,172 @@ def get(self, bin: int) -> Bin:
         :rtype: Bin
         """
         return self.bins[bin]
+
+
+class BinBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def buildBins(self, wheel: Wheel) -> None:
+        self.create_straight_bets(wheel)
+        self.create_horizontal_split_bets(wheel)
+        self.create_vertical_split_bets(wheel)
+        self.create_street_bets(wheel)
+        self.create_corner_bets(wheel)
+        self.create_line_bets(wheel)
+        self.generate_five_bets(wheel)
+        self.create_even_money_bets(wheel)
+        self.create_dozen_bets(wheel)
+        self.create_column_bets(wheel)
```

Comment:
`buildBins` is responsible to invoke helper methods that build/populate bins of wheel. The helper methods are not creating bets (in true sense) so method names like `create_...` does not seem appropriate. Instead `build_bins_for_x_bet` seems better choice as the name reflects the purpose.

Reply:
Done. Thanks.